examples/knowledge_tuning/translation_example/flows/translate_flow_knowledge.yaml

```
@@ -0,0 +1,57 @@
+        - "[End]"
+  gen_kwargs:
+    max_tokens: 512
+    temperature: 0.7
+
+- block_type: LLMBlock
+  block_config:
+    block_name: response_generation
+    model_id: "ibm-granite/granite-3.3-2b-instruct"
+    config_path: configs/generate_response.yaml
+    output_cols:
+      - response
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)